tikv-configuration-file.md

```diff
@@ -1269,6 +1269,25 @@ Configuration items related to BR backup.
 + Default value: `MIN(CPU * 0.75, 32)`.
 + Minimum value: `1`
 
+## cdc <span class="version-mark">New in v4.0.5</span>
+
+Configuration items related to TiCDC capturing changed data.
+
+### `min-ts-interval`
+
++ The interval at which timestamps are resolved.
++ Default value: `"1s"`
```

Comment:
Is the quotation mark correct?

Reply:
I searched similar default values in this doc like `"xs"` and all are wrapped in "".

Reply:
wrapped with `""` is correct, which means it is a string